src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -82,7 +82,10 @@ export const SchedulePageModal: FC<SchedulePageModalProps> = ({
         <li
           key={formatDate(day.dateDay.getTime())}
           className={`button ${idx === date ? "active" : ""}`}
-          onClick={() => setDate(idx)}
+          onClick={(e) => {
+            e.stopPropagation();
+            setDate(idx);
+          }}
```

Comment:
[FYI] `e.stopPropagation` is required here to stop the click event 'bubbling higher', and triggering the `hideEventSchedule` `onClick` on the `.schedule-dropdown-backdrop` `div` in `src/components/molecules/NavBar/NavBar.tsx`